### DISCUSSION TITLE
Failed to fetch postgresql xenial pgdg 404 Not Found

### DISCUSSION THREAD

**Discussion by duocdd92** [dokku/dokku#5367] [Q&A]
Hello everybody,

I have been using dokku for 3 years and it was working well. Until one day, it suddenly had a problem when pushing new source code to the remote server using the command `git push dokku@x.x.x.x:app master`

`Failed to fetch http://apt.postgresql.org/pub/repos/apt/dists/xenial-pgdg/main/binary-amd64/Packages 404 Not Found [IP: 72.32.157.246 80]`

When I open the website, I couldn't find the xenial-pgdg folder (it seems be removed). I tried to update to the APT URL to link to new package `bionic-pgdg` by updating a config  `deb http://apt.postgresql.org/pub/repos/apt bionic-pgdg main` to the following files
- `/etc/apt/sources.list.d/sources.list`
- `/etc/apt/sources.list.d/pgdg.list`
- `/etc/apt/sources.list.d/dokku.list`

But none of them work. Can anyone help me?
Thanks

Ubuntu 18.04.3 LTS
dokku 0.12.13

**Comment by duocdd92** [comment]
![index](https://user-images.githubusercontent.com/29083807/191911694-e39e9518-92f0-42b2-ade1-2a839669ecd3.png)

**Comment by josegonzalez** [comment]
You may wish to update your dokku installation and herokuish version via the following steps:

```
apt-get update
apt-get install dokku-update
dokku-update run --skip-rebuild
```

After that, using a newer repository based on bionic may help for pulling a newer version of that pgdg package. Xenial is no longer maintained, so I doubt that will work as folks remove package repositories.
